features/FxA-89-devices-view/README.md

```diff
@@ -0,0 +1,410 @@
+* Percentage of clicks on "missing or duplicate devices?" link.
+  * # of users that clicked it / # of users that saw device view
+* Percentage refresh rate
+  * # of users that clicked refresh / # of users that saw device view
+* Clicks on the "Connect..." button
+  * # of users that clicked it / # of users that saw the button
+* Number of mobile installs generated from the page.
+  * From mobile team metrics via adjust.com
+* Number of clicks on "I did not disconnect this device" in email
+  * From mobile team metrics via adjust.com
+
+
+## UX
+
+### Assets
+
+Use these icons for desktop, phone, and tablet device types:
+
+<img src="screenshot-of-device-icon-assets.png" />
+
+TODO: @ryanfeeley to provide raw asset files to add here.
+
+### Collapsed View
+
+A **Devices** section appears in the settings screen
+above the **Password** section:
+
+<img src="devices-view-collapsed.png" alt="Collapsed View" />
+
+### Fetching Devices View
+When the user clicks the **Show** button,
+and the server has to fetch the images,
+the **Show** button becomes disabled
+and is replaced with a spinning wheel
+(halved in size from full size spinning wheel).
+
+<img src="devices-view-fetching.png" alt="Fetching Devices View" />
+
+### Zero or one Device Views
+
+When the user expands the Device section,
+and has no devices,
+they are shown their current device
+but are able to download Firefox for iOS or Android
+and to refresh the list:
+
+<img src="devices-view-single-desktop-device.png" alt="Zero or one Device Views" />
+
+Links for Android and iOS go their respective stores.
+The mobile app placeholders should not be there
+if the user has a real mobile device
+attached to their account.
+
+### List View
+
+When the user expands the Devices section,
+the Firefox profiles that are syncing through this account
+are presented with the relative time they were “Last active”,
+with a "Disconnect..." button beside each.
+
+<img src="devices-view-populated.png" alt="List View" />
+
+Unlike in the first version of this feature,
+when devices are connected,
+we *should not* display “Firefox is available for Windows, OS X, etc…”.
+The reason for this is that we want to make sure
+we are not distracting the users
+when they are about to disconnect one of their devices.
+A distraction could result in them disconnecting the wrong device.
+
+When the user is viewing the device list
+from a web session on accounts.firefox.com,
+and is on a compatible Firefox browser
+that is not currently connected to Sync,
+we provide a "Connect..." button
+that will allow them to sign in to sync
+on that device.
+
+<img src="devices-view-current-web-session.png" alt="Zero or one Device Views" />
+
+### Disconnecting View
+
+When the user selects to disconnect a device,
+we overlay a modal confirmation dialog.
+This will minimize accidental clicks on "Disconnect..."
+and help us measure signals as to why a device is removed.
+
+In this view, we should not see
```

Comment:
IMHO it's mildly useful as motivation for the design choice, and low-cost to keep around.